file_handling/save.py

```diff
@@ -0,0 +1,96 @@
+import os
+from mantid import mtd
+from mantid.api import IEventWorkspace
+from mantid.simpleapi import \
+    CloneWorkspace, Rebin, ConvertToDistribution, DiffractionFocussing, \
+    SaveNexusProcessed, SaveAscii, DeleteWorkspace
+
+
+def save_banks(
+        InputWorkspace,
+        Filename,
+        Title,
+        OutputDir='./',
```

Comment:
Do we ever use `save_banks` with this default value. I'd suggest that we shouldn't have a default for this. `Binning` and `Grouping` defaults are fine